### PR TITLE
Video block has black background when upload is in progress or upload has failed

### LINKED ISSUE #1066: [iOS] Video block has black background when upload is in progress or upload has failed
When a video is in progress or upload has failed, video block has a black background.

It is a regression caused by this PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1046

Screen Shot 2019-06-04 at 12 46 41

Screen Shot 2019-06-04 at 12 47 28

### PR DESCRIPTION
**Pull request by marecar3**
Fixes : https://github.com/wordpress-mobile/gutenberg-mobile/issues/1066

It's regression caused by https://github.com/WordPress/gutenberg/pull/15857

To test :

Build WPiOS and WPAndroid against gutenberg-mobile PR : https://github.com/wordpress-mobile/gutenberg-mobile/pull/1068

### CODE REVIEW

**Comment by marecar3** [comment]
Hey @daniloercoli @pinarol, when I build WPAndroid against reverted changes from : https://github.com/WordPress/gutenberg/pull/15857, I can't reproduce this issue : https://github.com/wordpress-mobile/WordPress-Android/issues/9943

Is it maybe some Android platform version specific problem? I tried on 8.1 and 9.0 and it worked very well?

File: packages/block-library/src/video/edit.native.js [hunk 1]
```diff
@@ -202,9 +202,9 @@ class VideoEdit extends React.Component {
 								</View>
 							);
 
-							const videoStyle = {
+							const videoContainerStyle = {
```
Comment (pinarol) [inline packages/block-library/src/video/edit.native.js]: Could we revert renamings if they are not done for a specific reason? they are making it hard to distinguish the real fix

File: packages/block-library/src/video/edit.native.js [hunk 2]
```diff
@@ -214,14 +214,14 @@ class VideoEdit extends React.Component {
...
-											style={ videoStyle }
+											style={ { ...videoContainerStyle } }
 											source={ { uri: src } }
 											paused={ true }
 											muted={ true }
 										/>
 									}
 									{ ! showVideo &&
-										<View style={ { ...videoStyle, ...style.placeholder } }>
+										<View style={ { height: videoContainerHeight, width: '100%', ...style.placeholder } }>
```
Comment (pinarol) [inline packages/block-library/src/video/edit.native.js]: so the real fix here is discarding the backgroundColor prop from here right? and else is the same technically

File: packages/block-library/src/video/style.ios.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 @import "./style.native.scss";
 
-.video {
+.videoContainer {
```
Comment (pinarol) [inline packages/block-library/src/video/style.ios.scss]: also renaming here might not be necessary

File: packages/block-library/src/video/style.native.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 // @format
 
-.video {
+.videoContainer {
```
Comment (pinarol) [inline packages/block-library/src/video/style.native.scss]: and here

**Comment by daniloercoli** [comment]
> Is it maybe some Android platform version specific problem? I tried on 8.1 and 9.0 and it worked very well?

No @marecar3 - The issue was there on all versions of Android we tested. Android 9 and Android 7.1 were the ones I personally tested on real devices.

**Comment by pinarol** [comment]
@marecar3 @daniloercoli I have updated the fix to add a wrapper `View` around Video just to be able to give the same background color for both platforms. If you can give it a try that'd be great.

**Comment by marecar3** [comment]
Test on both iOS and Android and it's working! Thanks @pinarol for jumping in! :)

**Review by daniloercoli — APPROVED** [review]
LGTM!

**Pull request merged**